.github/workflows/verify.yml

```diff
@@ -119,7 +119,7 @@ jobs:
         working-directory: .github/utils
 
       # results of step should be visually examined in https://my.appoptics.com/
-      - name: Start Server & use Client againt Staging
+      - name: Start Server & use Client againt Production
```

Comment:
did you say "fix typos"?

againt -> against

Reply:
I'm blind...